src/main/scala/mesosphere/marathon/MarathonSchedulerService.scala

```diff
@@ -267,14 +267,18 @@ class MarathonSchedulerService @Inject() (
     // Actually we need to do the fresh twice, before the migration, to perform the migration on the current zk state and after
     // the migration to have marathon loaded the current valid state to the internal caches.
 
-    // refresh group repository cache
-    Await.result(groupManager.invalidateGroupCache(), Duration.Inf)
+    //    logger.info("RefreshCaches before migration")
+    //    // refresh group repository cache
+    //    Await.result(groupManager.invalidateGroupCache(), Duration.Inf)
```

Comment:
Maybe this should just be an invalidation. We probably want to adapt the comment above or verify that its claim is still true.

Reply:
I've cleaned that up and updated the comment.